dvc/command/get.py

```diff
@@ -33,7 +33,7 @@ def run(self):
 
 
 def add_parser(subparsers, parent_parser):
-    GET_HELP = "Download data from DVC repository."
+    GET_HELP = "Download files or directories from DVC repository."
```

Comment:
CC @jorgeorpinel

Reply:
Thanks @efiop, reviewing.

Reply:
@danihodovic hmmm does this actually support downloading directories tracked by Git?

Reply:
For now, let's start this with `Download/copy`.

Reply:
@jorgeorpinel "copy" doesn't make much sense to me. For the user it is all "download", since we _download_ a git repo and "copy" is an internal implementation detail.

Reply:
https://dvc.org/doc/command-reference/get#get
download is used in one form or another 13 times in that document, copy is only used one. For consistency I would use the same word throughout the docs.

Reply:
@efiop if the `path` is to a local file system repo, it's not a download, its a copy.

@danihodovic I need to check but I think other docs use "or copy". Correctness trumps consistency in any case. If anything let's try to add both words throughout the doc 🙂

Reply:
Actually now that I think about it, does it copy if it's from a local repo? Maybe not yet since this is open #2599. But still, to the user it looks more like a "copy" + we intend to implement that.

Reply:
Please also notice my other question here: "does this actually support downloading directories tracked by Git?" Thanks

Reply:
Me again. Please read my comments on GH because I have edited them. They weren't clear originally.

Reply:
> if the url is to a local file system repo, it's not a download, its a copy. Right?

@jorgeorpinel Well, it might be a reflink even, if it is supported(see copyfile in utils, which tries to create a reflink if it can). 🙂 My point is that `copy` is an implementation detail, and you can call `download` a "download" no matter if you are downloading from a local or a remote repo. But if you are certain that it is the best to explain it this way, let's keep it, I'm fine with that, maybe it is my "internals" bias kicking in. 🙂

Reply:
`dvc get` tries to just make a reflink from the cache of a external dvc repo if its located on the same file system?  I didn't know that! Well, reflinks are kind of copies anyway, I think (copy-on-write), manually done with `cp --reflink A B`.

> My point is that copy is an implementation detail, and you can call download a "download" no matter if you are downloading from a local or a remote repo.

You may be right. You also pointed out `For the user it is all "download"` which  I'm not sure is true. To me it seems wrong to call copying something from the same file system "download", **but it's an edge  case for sure** so maybe it's unnecessary to complicate adding the term "copy". In any case Dani already added `/copy` for now.

I'll open a separate issue to decide and finalize this? I'll mention you guys.

Reply:
My 2c on this. No need to complicate with `copy` in UI. I think the only place that is reasonable to mention copy is the Description part of the command reference for the `dev get` command.

Reply:
Extracted to #825